docs/2_3_common_states.adoc

```diff
@@ -737,7 +737,7 @@ In Scheduled Execution the <<intermediateUpdateTime>> should be ignored since in
 
 [[clocksTicked,`clocksTicked`]]
 * The <<clocksTicked>> parameter is only used in Scheduled Execution and is ignored in Co-Simulation.
-When <<clocksTicked,`clocksTicked == fmi3True`>>, it means that <<fmi3GetClock>> function must be called for gathering all <<Clock>> related information about ticking <<outputClock,`output clocks`>> and then activate the given model partitions accordingly.
+When <<clocksTicked,`clocksTicked == fmi3True`>>, it means that <<fmi3GetClock>> function must be called for gathering all <<Clock>> related information about ticking <<outputClock,`output clocks`>>, <<fmi3GetInterval>> must be called for all <<countdown>> <<Clock,Clocks>> and then activate the respective model partitions accordingly.
```

Comment:
This is also incorrect imho: "The importer takes the interval and schedules the model partitions." Only when it's the right time, is the model partition activated.